src/liblzma/check/crc32_loongarch.h

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: 0BSD
+
+///////////////////////////////////////////////////////////////////////////////
+//
+/// \file       crc32_loongarch.h
+/// \brief      CRC32 calculation with LoongArch optimization
+//
+//  Authors:    Xi Ruoyao
+//
+///////////////////////////////////////////////////////////////////////////////
+
+
+#ifndef LZMA_CRC32_LOONGARCH_H
+#define LZMA_CRC32_LOONGARCH_H
+
+#include <larchintrin.h>
+
+static uint32_t
+crc32_arch_optimized(const uint8_t *buf, size_t size, uint32_t _crc)
```

Comment:
We don't use variables starting with "_" since these type of identifiers are reserved. So perhaps `_crc` could be renamed as `crc_unsigned` or `crc_param` or something better that you can think of :)

Reply:
Done.